src/lib-methods.ts

```diff
@@ -0,0 +1,8 @@
+export {
```

Comment:
We could rename, idea here is that when we export from charmverse/core/X (except test folder), any imports will be supported in a client-side environment and not have prisma dependencies.

This is one annoying issue that led to using the "shared" folder

Reply:
we should call this "browser" then